tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
not sure this file should be here

Reply:
> please run prettier on all the modified files

What's the best way to do it? (or just git diff pipe prettier magic?)

Reply:
I think git diff will be the easiest

Reply:
done